src/prefs/ExtImportPrefs.cpp

```diff
@@ -120,7 +120,7 @@ void ExtImportPrefs::PopulateOrExchange(ShuttleGui & S)
          bool fillRuleTable = false;
          if (RuleTable == NULL)
          {
-            RuleTable = safenew Grid(S.GetParent(),EIPRuleTable);
+            RuleTable = safenew Grid(FormatterContext::EmptyContext(), S.GetParent(),EIPRuleTable);
```

Comment:
You could sneak AudacityProject& in here too, with a change to the factory function at the bottom of the file -- but only when it is given non-null.

Oh well, at least one use `EmptyContext()' is necessary then.